content/docs/tasks/telemetry/distributed-tracing/index.md

```diff
@@ -137,6 +137,29 @@ public Response bookReviewsById(@PathParam("productId") int productId,
 
 When you make downstream calls in your applications, make sure to include these headers.
 
+## Trace Sampling
+
+Istio is configured by default to capture all traces. For every time
+you access `/productpage` you should see a corresponding trace in the
+Jaeger dashboard. This is suitable for a test or low traffic mesh. For
+a high traffic mesh you may wish to lower the trace sampling
+percentage. This can be set in two ways:
+
+* On mesh setup, you may use the Helm option `pilot.traceSampling` to
+  set the percentage of trace sampling you want. See the
+  [Helm Install](/docs/setup/kubernetes/helm-install/) documentation for
+  details on setting options.
+* In a running mesh, you may edit the `istio-pilot` deployment and
+  change the environment variable by doing the below.
+
+  1. Run `kubectl -n istio-system edit deploy istio-pilot`. This will
```

Comment:
That command doesn't just open the text editor. If I am not mistaken, it opens the `istio-pilot configuration file on your text editor. Please rewrite the text to reflect the exact behavior the command causes. I also recommend placing the command within the {{< text bash >}}{{< /text >}} tags.

Reply:
Updated, had to move above the command to preserve the numbered list.